pure.zsh

```diff
@@ -582,6 +583,29 @@ prompt_pure_setup() {
 	# to be available, it was added in Zsh 5.3.
 	autoload -Uz +X add-zle-hook-widget 2>/dev/null
 
+	# most used colors
+	typeset -gA prompt_pure_colors
+	prompt_pure_colors=(
+		branch		242
+		error		red
+		exec_time	yellow
+		git_arrow	cyan
+		host		242
+		path		blue
+		root		white
```

Comment:
```suggestion
		root		reset
```

Use `reset` as default color, as https://github.com/sindresorhus/pure/pull/450 mentioned.

Reply:
I think it should be called `default`, I did a quick look through the man-pages and didn't find a `reset` color.

> Most terminals support the colours black, red, green, yellow, blue,  magenta,  cyan  and
white,  which can be set by name.  In addition. default may be used to set the terminal's default foreground colour.

Reply:
Changed to `default`.

Reply:
👍